crates/trie/sparse/src/state.rs

```diff
@@ -62,11 +62,19 @@ impl SparseStateTrie {
         self.revealed.get(account).is_some_and(|slots| slots.contains(slot))
     }
 
-    /// Returned mutable reference to storage sparse trie if it was revealed.
+    /// Returns mutable reference to storage sparse trie if it was revealed.
     pub fn storage_trie_mut(&mut self, account: &B256) -> Option<&mut RevealedSparseTrie> {
```

Comment:
this is confusing now, because `storage_trie_mut` returns a revealed trie too, but just behind an `Option` instead of a `Result`